src/_store/AppConfigProvider.js

```diff
@@ -23,9 +24,11 @@ export default class AppConfigProvider extends Component {
         language: PropTypes.string.isRequired,
         theme: PropTypes.string.isRequired,
     };
-
+    
     render() {
         const { children, language, theme } = this.props;
+        addLocaleData(localeData(locale(language)));
```

Comment:
Not entirely sure about `we are not loading on language change`
We can react on language change, either by creating the lifecycle method and call side-effects as needed, or hook on the actions that change language ~